### ISSUE TITLE
Use common mouse/keyboard combinations for zoom

### ISSUE THREAD

**Issue by fkaelberer** [desktop/desktop#3913]
### Description

Please use common keyboard mouse combinations for zooming:
`Ctrl`+`+` for zoom in
`Ctrl`+`-` for zoom out
`Ctrl`+`0` for zoom reset

Issue 1: Shortcut description in the menu is wrong
Issue 2: Numpad `+`/`-` is not supported as in Firefox, Chrome or IE
Issue 3: Mouse scrolling is not supported as in Firefox, Chrome or IE

### Version
**GitHub Desktop version:** 1.0.13
**OS version:** Windows 7

### Steps to Reproduce (Issue 1)
Open the View menu

**Expected behavior:** 
it shows
`Ctrl`+`+` for zoom in

**Actual behavior:**  
Menu screenshot: 
![grafik](https://user-images.githubusercontent.com/4624768/35694499-d8c0f2b0-0781-11e8-939e-3724c71dadba.png)

### Steps to Reproduce (Issue 2)
Press `Ctrl`+`NumPad-+`

**Expected behavior:** it zooms in as in Firefox, Chrome, or IE
**Actual behavior:** nothing happens

### Steps to Reproduce (Issue 3)
Hold `Ctrl` key and scroll with the mouse wheel

**Expected behavior:** it zooms in or out as in Firefox, Chrome, or IE
**Actual behavior:** nothing happens

**Comment by iAmWillShepherd** [comment]
Thanks for taking the time to open an issue! 🙇 

Just to make sure I understand correctly, the problem is that the zoom shortcut is labelled `Ctrl + =` and not `Ctrl +` and GitHub Desktop does not behave like Chrome, Firefox, Safari/IE?

**Comment by fkaelberer** [comment]
> Just to make sure I understand correctly, the problem is that the zoom shortcut is labelled Ctrl + = and  not Ctrl + and GitHub Desktop does not behave like Chrome, Firefox, or Safari/IE/Edge?

That's correct, but I'd rather say: It behaves differently than *any* other zoom-enabled software that I use on my computer, including MS Office, MS Visual Studio [Code], Qt Creator, Thunderbird, Acrobat Reader, Sumatra Pdf, IrfanView, Audacity, even Windows 10's built-in file explorer, in the sense that it does not support `ctrl`+mousewheel-zooming. Among the software titles which supports `ctrl`+`+/-`-zooming, all support also `ctrl`+`numpad+/-`.

**Comment by iAmWillShepherd** [comment]
Ok, just to make sure nothing is lost.

* `Ctrl/Cmd + =` should be changed to show `Ctrl/Cmd +`
* `Ctrl/Cmd + -` should  be changed to show `Ctrl/Cmd -`
* This short-cut should execute on the numeric keypad as well

**Comment by fkaelberer** [comment]
Not quite:
* `Ctrl/Cmd + =` should be changed to show `Ctrl/Cmd + 0`
* `Ctrl/Cmd + 0`, `Ctrl/Cmd + +` and `Ctrl/Cmd + -` short-cuts should execute on the numeric keypad as well
* Zooming with `Ctrl`+mousewheel should be supported

**Comment by iAmWillShepherd** [comment]
Thanks for the clarification @fkaelberer

**Comment by fkaelberer** [comment]
Hi @iAmWillShepherd, 
> ie, = and + are the same key.

that is only  true on some keyboard layouts. On my layout, `=` is the alternate character on the `0` key.

> Turns out Ctrl/Cmd + 0 and Ctrl/Cmd - already works on the numeric keypad, so the only change needed here is to make sure Ctrl/Cmd + works.

This does not work for me, tested on two Windows machines (Win 7 and 10).

> I think having the keyboard commands and the menu element is enough here. If you can offer a compelling case and some other applications that do this, please share.

As noted above, MS Office, MS Visual Studio [Code], Qt Creator, Thunderbird, Acrobat Reader, Sumatra Pdf, IrfanView, Audacity, even Windows 10's built-in file explorer do support `Ctrl`-mousewheel zooming. Did I miss any popular applications?

**Comment by j-f1** [comment]
On the Mac, Ctrl+scroll is often bound to zooming the entire screen.

**Comment by agisilaos** [comment]
Working on this issue now @iAmWillShepherd. Opening a PR soon with a tiny question. 😄

**Comment by agisilaos** [comment]
PR's ready #4244

**Comment by fkaelberer** [comment]
Related: https://github.com/electron/electron/issues/11310

**Comment by agisilaos** [comment]
@niik This is something I want to get back on and work on. But based on your comments on the PR (3 years ago) I'm baffled on how to move forward.

**Comment by jackashmore** [comment]
Upon re-reading the contribution guidelines, I missed the part saying to comment on an issue you're interested in if you intend to work on it... Oops. See PR above 😅
